### ISSUE TITLE
Bug: <ROOT CHUNK IS [random 4 letter txt] NOT FORM>

### ISSUE THREAD

**Issue by ctrl-alt-gif** [krzys-h/UndertaleModTool#901]
### Describe the bug

basically, I modded a Data.win of Mega Man Maker and changed some sprites and boss AI, and when I loaded the game, it worked perfectly. However, when I turned off the computer and turned it on again, the data.win completely breaks, showing this error:
![image](https://user-images.githubusercontent.com/62480771/167516949-1e442a2e-332c-4493-8c2d-f28a14bc77d4.png)

### Reproducing steps

Open the file
See error

### Setup Details

1.latest version (0.4.0.4)
2.Windows 10
3.Mega Man Maker v1.6.3 (megamanmaker.com, click "older versions and changelogs, scroll down to version 1.6.3 and click the download button")

**Comment by NyakoFox** [comment]
How did you turn off your computer? This file is 100% unrecoverable, the message says the root chunk is ÿÿÿÿ (where ÿ = 0xFF), meaning its most likely completely corrupted.
This is most likely not a problem with the tool itself, but the way you turned off your computer, where the OS's buffer didn't write to the disk before shutting off.

Did you hard cut the power to your computer?

**Comment by ctrl-alt-gif** [comment]
> 

No.

**Comment by ctrl-alt-gif** [comment]
It's okay though. my USB drive is probably corrupted, so im using the main disk until then.

**Comment by NyakoFox** [comment]
Either way, this is not a problem with the tool itself.

**Comment by Miepee** [comment]
Yes, this seems like a disk corruption error, not a bug of UTMT.
For future reference, it would be nicer if you wouldn't write in caps lock, or ping the developers of the tool.
